dnfdaemon-server/services/rpm/rpm.cpp

```diff
@@ -108,6 +114,25 @@ sdbus::MethodReply Rpm::list(sdbus::MethodCall & call) {
     return reply;
 }
 
+sdbus::MethodReply Rpm::distro_sync(sdbus::MethodCall & call) {
+    std::vector<std::string> specs;
+    call >> specs;
+
+    // read options from dbus call
+    dnfdaemon::KeyValueMap options;
+    call >> options;
+
+    // fill the goal
+    auto & goal = session.get_goal();
+    libdnf::GoalJobSettings setting;
+    for (const auto & spec : specs) {
+        goal.add_rpm_distro_sync(spec, setting);
+    }
```

Comment:
The current code supports only distro-syncing particular packages. It would be nice to add also support for `dnfdaemon-client distro_sync` - distro-sync all packages. There already is similar implementation in upgrade command.

Reply:
I updated the code, is this what you mean?

Reply:
Yes, thanks.